common/src/main/scala/com/gu/recipeasy/db/DB.scala

```diff
@@ -74,6 +75,18 @@ class DB(ctx: JdbcContext[PostgresDialect, SnakeCase]) {
     }
   }
 
+  def insertImages(images: List[Image]): Unit = {
```

Comment:
It would fail to post all the images but only for one article at a time. Might be worth tracking the number of articles that fail?